build/bin/vars

```diff
@@ -10,13 +10,13 @@ tmp_dir=${root_dir}/tmp
 # * pmm_release to contain only pre-release part (may be empty): 'alpha3'.
 pmm_branch=$(git rev-parse --abbrev-ref HEAD)
 full_pmm_version=$(cat ${root_dir}/VERSION)
-if [[ ${pmm_branch} != 'PMM-2.0' ]]; then
+if [[ ${pmm_branch} != "release"* ]]; then
```

Comment:
I think you should use `or` condition here, we also use specific version for dev-latest builds which are build from `PMM-2.0`

Reply:
why should we have specific version for the PMM-2.0? BTW here I put some table: https://jira.percona.com/browse/PMM-7954?focusedCommentId=282194&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel

Reply:
I am still not sure from my point of view, it should be fine building dev-latest also with a specific version, but your call, this change would also mean changing package-testing and test suite.

Reply:
I don't mind to change, just need to understand why and where we have this dependency.
What dependency we have for the testing ?

Reply:
so for test suite, we don't need any changes because we don't do strict version checks due to FB's and for package-testing, I have already sent a [PR](https://github.com/Percona-QA/package-testing/pull/76/files) it contains changes for repo's also.

Reply:
I added it anyway, to not break other things.